unittests/AST/ASTImporterTest.cpp

```diff
@@ -55,22 +57,31 @@ const StringRef DeclToVerifyID = "declToVerify";
 // Common base for the different families of ASTImporter tests that are
 // parameterized on the compiler options which may result a different AST. E.g.
 // -fms-compatibility or -fdelayed-template-parsing.
-struct ParameterizedTestsFixture : ::testing::TestWithParam<ArgVector> {
+class LanguageSpecificTest : public ::testing::Test {
```

Comment:
This seems like a better name we had previously, I also like the abstraction on `getExtraArgs`.

Reply:
But it is not really language specific but rather compiler option specific. So perhaps `CompilerOptionSpecificTest` is more descriptive.